packages/components/src/StatusIndicator/StatusIndicator.css

```diff
@@ -0,0 +1,8 @@
+.statusIndicator {
+  width: var(--statusLabel-icon-diameter);
+  height: var(--statusLabel-icon-diameter);
+  margin-top: var(--space-minuscule);
```

Comment:
i'd actually take this margin out as that would create so much pain for us when you try and use this component where a parent is vertically aligning to the middle and this is offset by 1px down.

I get why this is here tho because of status label so i'd find a way to preserving old styling and alignment on StatusLabel while keeping StatusIndicator free of opinions about spacing.